Libs/Widgets/ctkErrorLogWidget.h

```diff
@@ -49,6 +49,11 @@ class CTK_WIDGETS_EXPORT ctkErrorLogWidget : public QWidget
   /// \sa ctkErrorLogModel::ColumnsIds
   Q_INVOKABLE void setColumnHidden(int columnId, bool hidden) const;
 
+  /// Optimize the widget grid layout for vertical or horizontal view.
+  /// Default is vertical.
+  Q_INVOKABLE void setLayoutOrientation(Qt::Orientation orientation);
```

Comment:
I suggest calling `this->setLayoutOrientation(Qt::Vertical)` from `ctkErrorLogWidgetPrivate::init()`

Reply:
Instead of using `Q_INVOKABLE`, consider adding a `Q_PROPERTY` and consider moving `setLayoutOrientation` as a slot.

Reply:
yes, done in https://github.com/commontk/CTK/pull/1100/commits/94c394c37c1f9ac88a40c1577bbf593524d1ef12